creator-node/scripts/start.sh

```diff
@@ -67,14 +67,15 @@ fi
 
 # index.js runs multiple processes using cluster. Starts as primary since process.env.NODE_UNIQUE_ID=undefined
 if [[ "$devMode" == "true" ]]; then
+    export clusterModeEnabled="false"
     if [ "$link_libs" = true ]; then
         cd ../audius-libs
         npm link
         cd ../app
         npm link @audius/sdk
-        npx nodemon --exec 'node --inspect=0.0.0.0:${debuggerPort} --require ts-node/register src/index.ts' --watch src/ --watch ../audius-libs/dist/index.cjs.js | tee >(logger)
+        npx nodemon --verbose --exec 'node --inspect=0.0.0.0:${debuggerPort} --require ts-node/register src/index.ts' --watch src/ --watch ../audius-libs/dist/index.cjs.js | tee >(logger)
```

Comment:
Keep the `verbose` or was it just for testing?

Reply:
it was for debugging to see if restarts were caused by something else reloading. I kinda like keeping it tho because it says which file was changed so we can confirm if it's using linked libs without checking the startup script. doesn't actually add much log spam and it's only local